### ISSUE TITLE
In Angular demo how i can enable checkBoxSelection and pagination , deferredRenderring, inlineEditing ?

### ISSUE THREAD

**Issue by SharpieX** [helion3/inspire-tree#106]
is it possible to use default renderar so that i can give these properties to tree config ? this doesnt seems working cause we have overridden HTML in directives defined in ngInspiretree.js

$scope.treeOptions = {
data: treeColumns,
dom: {
deferredRendering: true
},
pagination: {
limit:30
},
editable:true,
}
P.S. -New to Angular

**Comment by viveleroi** [comment]
Please use gitter for support, issues are for feature requests or for bug reports. However, I'll still always reply here too, so no one will be ignored.

Answering your question - you can't.

InspireTree comes with a native rendering system, so the Angular 1, Angular 2, React, and Inferno custom DOM implementation demos are *basic*. They're just for example, proving that you can leave off our native renderer and implement your own. The caveat to all that is that when you implement your own renderer, you would have to also implement all of the DOM-specific stuff too.

**Comment by viveleroi** [comment]
Also, I'm not clear on your second question in body of this issue, you can't use both the default renderer *and* a custom one like Angular at the same time.

However, you can use the default renderer and Angular at the same time though, you just wouldn't use Angular for anything *except* initializing InspireTree on an element.

**Comment by SharpieX** [comment]
my data set is pretty huge like root have 8 nodes and count of  these children of these nodes are approx 4000 so I can see while rendering lag , so I wanted to use deferredRenderring in treeOptions to enable lazy load in angular demo you provided

**Comment by viveleroi** [comment]
Deferred rendering/loading will require the native DOM library. Deferred loading would be much easier to implement with a custom DOM, but still won't work automatically.

However, neither are available in a release yet.

**Comment by SharpieX** [comment]
thanks for a quick reply 
ah I see any idea when we can see it action :)

**Comment by viveleroi** [comment]
I'm rewriting the renderer for 1.11 but once that works is done, and I've written more tests, it should be ready. That will include all of the recent deferrals and other work.

**Comment by SharpieX** [comment]
thanks awaiting for feature
